iOS-NOTTODO/iOS-NOTTODO/Presentation/Auth/AuthViewController.swift

```diff
@@ -241,21 +241,61 @@ extension AuthViewController {
                     guard let userId = result?.data?.userId else { return }
                     KeychainUtil.setAccessToken(accessToken)
                     Amplitude.instance().setUserId(userId)
-                    self?.presentToHomeViewController()
+                    self?.checkNotificationSettings()
                     
                 }
             }
         }
     }
     
     func presentToHomeViewController() {
-        if let window = view.window?.windowScene?.keyWindow {
-            let tabBarController = TabBarController()
-            let navigationController = UINavigationController(rootViewController: tabBarController)
-            navigationController.isNavigationBarHidden = true
-            window.rootViewController = navigationController
+        DispatchQueue.main.async {
+            if let windowScene = UIApplication.shared.connectedScenes.first as? UIWindowScene,
+               let window = windowScene.windows.first {
+                let tabBarController = TabBarController()
+                let navigationController = UINavigationController(rootViewController: tabBarController)
+                navigationController.isNavigationBarHidden = true
+                window.rootViewController = navigationController
+                window.makeKeyAndVisible()
```

Comment:
#193 액세스 토큰 로직처리하면서 SceneDelgate에 `changeRootViewControllerTo` 메서드를 만들어뒀어여! #193 머지 후에 아래와 같이 변경해도 좋을 것 같아욤 :)
```suggestion
            SceneDelegate.shared?.changeRootViewControllerTo(TabBarController())
```

Reply:
헉 감사합니다!! #193 머지 후 수정하겠습니다 :)